src/QA/word_embed.py

```diff
@@ -10,7 +10,16 @@ class WordEmbed:
     def glove_embed_sent(sentence):
         embeds = []
         for word_tup in sentence:
-            embeds.append(glo100model[word_tup[0].lower()])
+            # TODO - some word_tups are empty?
+            if word_tup == "":
+                embeds.append(glo100model["empty"])
+            # TODO - choose a more appropriate way of handling this
+            lowered = word_tup[0].lower()
+            print(lowered)
+            if lowered in glo100model:
+                embeds.append(glo100model[lowered])
+            else:
+                embeds.append(glo100model["unknown"])

```

Comment:
Can you explain what this function is meant to be doing?
I dont get why we are grabbing word_tup[0], isnt that just one character?
i was getting some errors here while trying to embed on the train data set.

Reply:
@ndeclarke

Reply:
Yes, it shouldn't have been word_tup[0], and I have changed that in my changes this morning
Originally we had a word_tup of (word, answer-label), but when I changed the way the tokenisation was working I separated the two of those. I didn't realise my error until I was fixing the errors last night.